src/Sql/Sql/Database/Cmdlet/NewAzureSqlDatabase.cs

```diff
@@ -198,14 +205,38 @@ public class NewAzureSqlDatabase : AzureSqlDatabaseCmdletBase<AzureSqlDatabaseCr
         /// </summary>
         [Parameter(Mandatory = false,
             HelpMessage = "The Backup storage redundancy used to store backups for the SQL Database. Options are: Local, Zone and Geo.")]
-        [ValidateSet("Local", "Zone", "Geo")]
+        [ValidateSet("Local", "Zone", "Geo", IgnoreCase = false)]
```

Comment:
Change validate set IgnoreCase from true to false is a breaking change. Is it a bug fix? Will it report error from server when case mismatched?

Reply:
This is a newly introduced paramater (as part of [PR 12899](https://github.com/Azure/azure-powershell/pull/12899)); @dingmeng-xue and I discussed this.  We introduced this to match the MI user experience ([PR 12849](https://github.com/Azure/azure-powershell/pull/12849)).